get-started/create-run.qmd

```diff
@@ -0,0 +1,80 @@
+---
+title: "Create and Run"
+---
+
+When you [install shiny](get-started/install.qmd), you will have access to the `shiny` command line interface (CLI).
+You can use this interface to help you create and run your Shiny applications.
+
+## Create a Shiny application
+
+The best way to create a new Shiny app is with the `shiny create` command line interface.
+This command asks you a series of questions about what kind of app you want to create,
+and then provides all the boilerplate code you need to get started with a working app.
+
+```bash
+shiny create
+```
+
+![Running the shiny create command from a terminal](/docs/assets/shiny-create.mp4){class="img-shadow"}
+
+::: callout-tip
+### Copy/paste examples
+
+If you find an example on this site that you want to run or edit locally,
+you can use the following command to get a basic app template,
+then copy and paste the code from the example into the template.
+
+```bash
+shiny create --template basic-app -m express
+```
+:::
```

Comment:
Instead of this callout, I think it'd be better to link out to the template gallery here